### DISCUSSION TITLE
Hi!

### DISCUSSION THREAD

**Discussion by jongfeel** [humansof42seoul/Humansof42-v2#6] [General]
안녕하세요 김종필입니다.

humansof42seoul organization에 초대 받았을 때
open project repository를 알려주지 않아서 여기서 하는지는 방금 알았습니다.

아직 개발 툴 + 환경 세팅 중인데
진행하면서 의사 소통을 지속적으로 잘 해 봤으면 좋겠습니다.

Issues 탭은 주로 task 위주로 작성하는 곳이라 첫 인사는 여기서 해 봅니다.

**Comment by jwon42** [comment]
@joho3116 @42hm @ohdair 
멘토님 어제 감사했습니다! :)
